highfidelity/hifi
# Image entity has correct orientation and is collisionless

## Test Plan
1. Drag and drop an image into interface
2. The image should be facing you, and you should be able to walk through it
3. Find an image in the asset browser (.jpg or .png) and click "Add To World"
4. That image should also be facing you, and you should be able to walk through it
5. Go into Create mode, and add an image
6. Switch the Image URL to this: https://i.ytimg.com/vi/4PDQ1gziLL8/maxresdefault.jpg
7. The image should now have a great picture of a doge'n pupper
8. Walk through the doge, dont be skared if he does a lick
